db/remote/db.go

```diff
@@ -21,7 +21,7 @@ type DB struct {
 }
 
 func New(rawURL string, ctx context.Context, log utils.SimpleLogger, opts ...grpc.DialOption) (*DB, error) {
-	grpcClient, err := grpc.Dial(rawURL, opts...)
+	grpcClient, err := grpc.NewClient(rawURL, opts...)
```

Comment:
logic changed a bit: previously `grpc.Dial(...)` was connecting to rawURL, with`gprc.NewClient(...)` no connection is made until there is some activity on `grpcClient`. 

Which is fine I think